.eslintrc.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
         '@typescript-eslint/explicit-module-boundary-types': 'off',
         '@typescript-eslint/no-empty-function': 'off',
         '@typescript-eslint/no-inferrable-types': 'off',
+        '@typescript-eslint/ban-ts-comment': 'off',
```

Comment:
I think this is fine because it makes sure `data-attr` can be set on `okButtonProps`, but as it can have unintended consequences elsewhere, I'd like to get @mariusandra's POV here too.

Reply:
This doesn't change anything, just disables a warning when using `@ts-expect-error`. It's a directive we should use very rarely and only with sufficient rationale (as is here).